### ISSUE TITLE
Input for Cityscapes

### ISSUE THREAD

**Issue by jthsieh** [junyanz/pytorch-CycleGAN-and-pix2pix#199]
Most datasets used in CycleGAN are image to image datasets, but Cityscapes is label to image. I'm wondering what the input is. Is it the color map or the label map? If it's the label map, then do you change the generator's last output to be a N-class softmax?

**Comment by junyanz** [comment]
We are using the color map (RGB 3 channels) in CycleGAN and pix2pix.
